packages/microcosm/src/subject.js

```diff
@@ -160,12 +164,13 @@ export class Subject {
   }
 
   static hash(obj: *): Subject {
+    let subject = new Subject()
+
     if (getObservable(obj)) {
-      return obj
+      obj.subscribe(subject)
+      return subject
     }
```

Comment:
This change is important because the return value was inconsistent. I'm a little bummed Flow didn't catch this. We can't return an `Observable` because they play by different rules than `Subject`